docs/skill_development/user-interaction/confirmations.md

```diff
@@ -0,0 +1,15 @@
+---
+description: >-
+  Confirmations are used to verify that the input from the User was understood
+  correctly. These may be verbal or non-verbal.
+---
+
```

Comment:
Note that neon has a common practice of enabling confirmation that an intent handler is doing something when the action might take some time
```
        if get_user_prefs(message)['response_mode'].get('hesitation'):
            self.speak_dialog("check_updates")
```